modules/@apostrophecms/ui/ui/apos/scss/mixins/_theme_mixins.scss

```diff
@@ -0,0 +1,42 @@
+$font-default: -apple-system, system-ui, 'BlinkMacSystemFont', 'Segoe UI',
```

Comment:
It would be clearer to have a separate variables file. We could probably clean up files by moving more variables scattered about into that file and then get more of the disparate actual SCSS mixins together as well.

Reply:
They could be different files if that makes sense, but imported into a main mixins file to get into the full import.

Reply:
I think that might require some deliberation among the frontend crew. Can we accept this PR as an iterative improvement and open a new ticket for that?

Reply:
Yep.